fe/fe-core/src/main/java/com/starrocks/analysis/Delimiter.java

```diff
@@ -31,29 +31,97 @@ public static String convertDelimiter(String originStr) throws AnalysisException
             throw new AnalysisException("Delimiter cannot be empty or null");
         }
 
-        if (originStr.toUpperCase().startsWith("\\X") || originStr.toUpperCase().startsWith("0X")) {
-            String hexStr = originStr.substring(2);
-            // check hex str
-            if (hexStr.isEmpty()) {
-                throw new AnalysisException("Invalid delimiter '" + originStr + ": empty hex string");
-            }
-            if (hexStr.length() % 2 != 0) {
-                throw new AnalysisException("Invalid delimiter '" + originStr + ": hex length must be a even number");
-            }
-            for (char hexChar : hexStr.toUpperCase().toCharArray()) {
-                if (HEX_STRING.indexOf(hexChar) == -1) {
-                    throw new AnalysisException("Invalid delimiter '" + originStr + "': invalid hex format");
-                }
+        StringWriter writer = new StringWriter();
+
+        for (int i = 0; i < originStr.length(); i++) {
+            char ch = originStr.charAt(i);
+            boolean outputOneChar = true;
```

Comment:
What's the purpose of this pull request? It's to support \n to be column separator>

Reply:
yes